tests/test_tasks.py

```diff
@@ -0,0 +1,108 @@
+from rest_framework import status
+from .common import EndpointTestCase
+
+
+class TestTasksGet(EndpointTestCase):
```

Comment:
It'd be more uniform with other tests to define `TestTasks` and `TestSingleTask`.
Otherwise we can change the convention in other tests, depending on how we want to group them.

Reply:
Ok I have changed this after discussion in #181 and other next PRs